app/pages/reservationEdit/ReservationEditPageSelector.js

```diff
@@ -29,7 +29,13 @@ const resourceSelector = createSelector(
   (resources, resourceId) => resources[resourceId]
 );
 
+const isFetchingSelector = createSelector(
+  resourceSelector,
+  resource => !resource
+);
+
```

Comment:
Shieeet, I actually used reservationGetIsActiveSelector before and then for some reason did a fixup which removed that. So I didn't A) use my brain and B) test it after the change :D But good that you noticed! It's fixed now and also uses `resourcesGetIsActiveSelector` as you suggested.